index.html

```diff
@@ -14,4 +14,6 @@
         <footer class="grid-item" id="grid-footer">footer</footer>
```

Comment:
Hi Rik, volgens mij is het de bedoeling om na de <div> de volgende elementen zoals header, aside e.d. 1 tab in te springen (in dit geval zouden ze recht onder elkaar komen. Dit kun je ook aan het einde van het coderen automatisch doen door ctrl+alt+l (voor Windows)

Reply:
Ik zie nu dat wanneer ik de code naar Webstorm kopieer, de elementen wel netjes onder elkaar staan.